apple/RCTMarkdownUtils.mm

```diff
@@ -29,6 +29,12 @@ - (NSAttributedString *)parseMarkdown:(nullable NSAttributedString *)input
     if (input == nil) {
       return nil;
     }
+    
+    // Style and parser Id may not be initialized immediately due to the order of mount instructions
+    // props update will be executed after the view hierarchy is initialized.
+    if (_markdownStyle == nil || _parserId == nil) {
+      return nil;
+    }
```

Comment:
This is a bit unfortunate :(

Also
```suggestion
    // `_markdownStyle` and `_parserId` may not be initialized immediately due to the order of mount instructions
    // props update will be executed after the view hierarchy is initialized.
    if (_markdownStyle == nil || _parserId == nil) {
      return nil;
    }
```

Reply:
[7e4ac49](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/7e4ac498cefaf03505210b31a07a73bcce7fe060)